[2023-12-12 01:20:44 +0000] [11070] [INFO] Starting gunicorn 21.2.0
[2023-12-12 01:20:44 +0000] [11070] [INFO] Listening at: http://0.0.0.0:8000 (11070)
[2023-12-12 01:20:44 +0000] [11070] [INFO] Using worker: sync
[2023-12-12 01:20:44 +0000] [11071] [INFO] Booting worker with pid: 11071
[2023-12-12 01:20:44 +0000] [11071] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/ubuntu/py_env/lib/python3.10/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'flask_app'
[2023-12-12 01:20:44 +0000] [11071] [INFO] Worker exiting (pid: 11071)
[2023-12-12 01:20:44 +0000] [11070] [ERROR] Worker (pid:11071) exited with code 3
[2023-12-12 01:20:44 +0000] [11070] [ERROR] Shutting down: Master
[2023-12-12 01:20:44 +0000] [11070] [ERROR] Reason: Worker failed to boot.
